.babelrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "sourceType": "unambiguous",
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": 100,
+          "safari": 15,
+          "firefox": 91
+        }
+      }
+    ],
+    "@babel/preset-typescript",
+    "@babel/preset-react"
+  ],
+  "plugins": []
+}
```

Comment:
Missing newline at end of file

Reply:
https://github.com/inrupt/solid-ui-react/pull/901/commits/01865653bae86614ceaec0443aec073e2866f320